libminifi/test/aws-tests/AWSCredentialsServiceTest.cpp

```diff
@@ -53,14 +53,14 @@ TEST_CASE_METHOD(AWSCredentialsServiceTestAccessor, "Test expired credentials ar
   auto aws_credentials_impl = std::static_pointer_cast<minifi::aws::controllers::AWSCredentialsService>(aws_credentials_service->getControllerServiceImplementation());
 
   // Check intial credentials
-  REQUIRE(aws_credentials_impl->getAWSCredentials().GetAWSAccessKeyId() == "key");
-  REQUIRE(aws_credentials_impl->getAWSCredentials().GetAWSSecretKey() == "secret");
-  REQUIRE(!aws_credentials_impl->getAWSCredentials().IsExpired());
+  REQUIRE(aws_credentials_impl->getAWSCredentials()->GetAWSAccessKeyId() == "key");
```

Comment:
I would put a `REQUIRE(aws_credentials_impl->getAWSCredentials())` before this line, so we get a test failure rather than undefined behavior if the credentials do not exist.